tensorrt_llm/quantization/quantize_by_modelopt.py

```
@@ -1,4 +1,4 @@
-# SPDX-FileCopyrightText: Copyright (c) 2022-2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-FileCopyrightText: Copyright (c) 2022-2026 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Ruff (0.15.12)

[warning] The file is executable but no shebang is present

(EXE002)